test/kpod_pause.bats

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bats
+
+load helpers
+
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT $STORAGE_OPTS"
+function teardown() {
+    cleanup_test
+}
+
+@test "pause a bogus container" {
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} pause foobar
+    echo "$output"
+    [ "$status" -eq 1 ]
+}
+
+@test "unpause a bogus container" {
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} unpause foobar
+    echo "$output"
+    [ "$status" -eq 1 ]
+}
+
+@test "pause a running container by id" {
+    start_crio
+    run crioctl pod run --config "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    pod_id="$output"
+    run crioctl ctr create --config "$TESTDATA"/container_config.json --pod "$pod_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr_id="$output"
+    run crioctl ctr start --id "$ctr_id"
+    echo "$output"
+    id="$output"
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} pause "$id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} unpause "$id"
+    echo "$output"
```

Comment:
is there anything we could do test wise here to make sure the container was up and functional again after the unpause?

Reply:
No idea.  Do we have wget tests that would fail when paused and succeed when running.

Reply:
If you had a trivial shell (busybox?) you could throw up a trivial script that sleeps, then echos to a file on a volume mount, and check for the presence of the file after the test.

Reply:
I took the chickens way out and just did a 'kpod ps' here to see if the container was up and running afterwards.

Reply:
I may come back and add to it, had troubles trying to figure out quickly how to do @mheon 's suggestion